src/main/java/frc/robot/subsystems/CoralArm.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class CoralArm extends SubsystemBase {
+    private DoubleSolenoid piston;
+    private DoubleSolenoid piston2;
+    private DigitalInput extendlimitswitch;
+    private DigitalInput retractlimitswitch;
```

Comment:
```suggestion
    private final DigitalInput retractedLimitSwitch
```
finalize where possible. name should be camalCase